.github/workflows/docs.yaml

```diff
@@ -15,9 +15,7 @@ jobs:
         uses: actions/checkout@v3
 
       - name: Install Node
-        uses: actions/setup-node@v3
-        with:
-          node-version: 16
```

Comment:
We should update to the latest known working version. Node is also v23 will also not compatible currently.

Reply:
Looks like that should be v22 right now.